src/plugin.json

```diff
@@ -73,6 +77,226 @@
+        ]
+      },
+      "grants": ["Admin", "Editor"]
+    },
+    {
+      "role": {
+        "name": "Alerts reader",
+        "description": "Read alerts in the Synthetic Monitoring app",
+        "permissions": [
+          { "action": "plugins.app:access", "scope": "plugins:id:grafana-synthetic-monitoring-app" },
+          { "action": "grafana-synthetic-monitoring-app:read" },
+
+          { "action": "grafana-synthetic-monitoring-app.alerts:read" }
+        ]
+      },
+      "grants": ["Admin", "Editor", "Viewer"]
+    },
+    {
+      "role": {
+        "name": "Alerts writer",
+        "description": "Create, edit and delete alerts in the Synthetic Monitoring app",
+        "permissions": [
+          { "action": "plugins.app:access", "scope": "plugins:id:grafana-synthetic-monitoring-app" },
+          { "action": "grafana-synthetic-monitoring-app:read" },
+          { "action": "grafana-synthetic-monitoring-app:write" },
+
+          { "action": "grafana-synthetic-monitoring-app.alerts:edit" },
+          { "action": "grafana-synthetic-monitoring-app.alerts:read" },
+          { "action": "grafana-synthetic-monitoring-app.alerts:delete" }
+        ]
+      },
+      "grants": ["Admin", "Editor"]
+    },
+    {
+      "role": {
+        "name": "Thresholds reader",
+        "description": "Read thresholds in the Synthetic Monitoring app",
+        "permissions": [
+          { "action": "plugins.app:access", "scope": "plugins:id:grafana-synthetic-monitoring-app" },
+          { "action": "grafana-synthetic-monitoring-app:read" },
+
+          { "action": "grafana-synthetic-monitoring-app.thresholds:read" }
+        ]
+      },
+      "grants": ["Admin", "Editor", "Viewer"]
+    },
+    {
+      "role": {
+        "name": "Thresholds writer",
+        "description": "Read and edit thresholds in the Synthetic Monitoring app",
+        "permissions": [
+          { "action": "plugins.app:access", "scope": "plugins:id:grafana-synthetic-monitoring-app" },
+          { "action": "grafana-synthetic-monitoring-app:read" },
+          { "action": "grafana-synthetic-monitoring-app:write" },
+
+          { "action": "grafana-synthetic-monitoring-app.thresholds:edit" },
+          { "action": "grafana-synthetic-monitoring-app.thresholds:read" }
+        ]
+      },
+      "grants": ["Admin", "Editor"]
+    },
+    {
+      "role": {
+        "name": "Access tokens writer",
+        "description": "Create and delete access tokens in the Synthetic Monitoring app",
+        "permissions": [
+          { "action": "plugins.app:access", "scope": "plugins:id:grafana-synthetic-monitoring-app" },
+          { "action": "grafana-synthetic-monitoring-app:read" },
+          { "action": "grafana-synthetic-monitoring-app:write" },
+
+          { "action": "grafana-synthetic-monitoring-app.access-tokens:create" },
+          { "action": "grafana-synthetic-monitoring-app.access-tokens:delete" }
+        ]
+      },
+      "grants": ["Admin", "Editor"]
```

Comment:
Do we want Editors to create access tokens? This feels like an admin action to me.

Reply:
Yeah I think it makes more sense to be restricted to admins. Changed it.